Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fpd"
-version = "2.20.3"
+version = "2.20.0"
```

Comment:
This seems a bit odd...

Reply:
The previous version was `2.19.3`, and the automated bump pr didn't reset the patch version number before changing the minor version number